content/docs/command-reference/add.md

```diff
@@ -145,28 +145,31 @@ not.
   [pattern](https://docs.python.org/3/library/glob.html) specified in `targets`.
   Shell style wildcards supported: `*`, `?`, `[seq]`, `[!seq]`, and `**`
 
-- `--external` - allow `targets` that are outside of the DVC repository. See
+- `-o <path>`, `--out <path>` - destination `path` inside the workspace to place
+  a data target (instead of using the current working directory). Directories
+  specified in the path will be created by this command. Note that this can be
+  used [with an external cache](#straight-to-cache) to avoid using the local
+  file system.
+
+- `--to-remote` - allow a target outside of the DVC repository (e.g. an S3
+  object, SSH directory URL, file on mounted volume, etc.) but don't move it
+  into the workspace, nor cache it. [Store a copy](#straight-to-remote) on a
+  remote instead (the default one unless `-r` is specified). Use `dvc pull` to
+  get the data locally later.
```

Comment:
remove locally? it can be on a remote machine?

Reply:
Similar to `-o`, it can be useful when you need to add large data and don't have enough space on machine/volume you run this command on (e.g. to bootstrap the project, to update some data .... -

Reply:
Removed `locally` and extended the motivation with "to skip the local file system". The link has all the details but if you prefer I can expand on this even more (may need a 2nd p).